packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -645,11 +660,11 @@ class _SnackBarState extends State<SnackBar> {
                     ),
                   ),
                 ),
-                if(!actionLineOverflow) ...maybeActionAndIcon,
-                if(actionLineOverflow) SizedBox(width: snackBarWidth*0.4),
+                if(!willOverflowAction) ...maybeActionAndIcon,
+                if(willOverflowAction) SizedBox(width: snackBarWidth*0.4),
               ],
             ),
-            if(actionLineOverflow) Padding(
+            if(willOverflowAction) Padding(
```

Comment:
[Sorry to keep harping on the trivial stuff]. This is actually hard to read due to the indentation.

```dart
if (willOverflowAction) Padding(
  padding: const EdgeInsets.only(bottom: _singleLineVerticalPadding),
  child: Row(mainAxisAlignment: MainAxisAlignment.end, children: maybeActionAndIcon),
),
```